### PR TITLE
Change wordings and a bit of style for p2 creation

### PR DESCRIPTION
**Pull request by naxoc**
#### Changes proposed in this Pull Request
This changes some slightly confusing wording and a tiny bit of styling on the p2 creation form. 

See 4529-gh-Automattic/p2 and 4531-gh-Automattic/p2

#### Testing instructions
Go to `http://calypso.localhost:3000/start/p2/p2-site` and look at the changes there. You'll see changes to the "Hooray" screen once you submit the form.

Before:
Screen Shot 2022-01-28 at 16 49 27

After:

Screen Shot 2022-01-28 at 16 43 20

Related to #

### CODE REVIEW

**Review by nefeline — APPROVED** [review]
Confirmed the changes are in place ✨! Way more intuitive description indeed 👍

**Comment by evilluendas** [comment]
I like it! One small tweak I'd do: since this is isn't the final design for the renewed signup flow (for example, the P2 icon in the new one will be on the corner of the screen) and the next screen is still aligned to the left, what if we align the header and intro to the left too? Like this:

image


That way this part of the flow is more consistent until we replace it altogether.

I think I'd also try removing the "P2" part on the second field's label so it's `Choose an address for your workspace`, given that we're using just "workspace" the previous couple of times. Aside from that, I'd say 🚢 !

File: client/signup/p2-processing-screen/index.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ function P2SignupProcessingScreen() {
 			</div>
 
 			<div className="p2-processing-screen__text">
-				{ translate( '{{h2}}Hooray!{{/h2}} {{p}}Your new P2 is almost ready.{{/p}}', {
+				{ translate( '{{h2}}Hooray!{{/h2}} {{p}}Your new P2 workspace is almost ready.{{/p}}', {
```
Comment (a8ci18n) [inline client/signup/p2-processing-screen/index.jsx:14]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/signup/steps/p2-site/index.jsx [hunk 2]
```diff
@@ -404,7 +404,7 @@ class P2Site extends Component {
 					className="p2-site__validation-site"
 				>
 					<FormLabel htmlFor="site-address-input">
-						{ this.props.translate( 'Choose an address for your P2 workspace' ) }
+						{ this.props.translate( 'Choose an address for your workspace' ) }
```
Comment (a8ci18n) [inline client/signup/steps/p2-site/index.jsx:407]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/signup/steps/p2-site/index.jsx [hunk 3]
```diff
@@ -461,11 +461,13 @@ class P2Site extends Component {
 	render() {
 		return (
 			<P2StepWrapper
+				className="p2-site__create"
 				flowName={ this.props.flowName }
 				stepName={ this.props.stepName }
 				positionInFlow={ this.props.positionInFlow }
-				headerText={ this.props.translate(
-					'Share, discuss, review, and collaborate across time zones, without interruptions.'
+				headerText={ this.props.translate( 'Create a workspace' ) }
```
Comment (a8ci18n) [inline client/signup/steps/p2-site/index.jsx:468]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Create a new workspace' )` *ES Score: 6*

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7123737

Thank you @naxoc for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7123737).

**Pull request merged**